### DISCUSSION TITLE
Mysql dump problem

### DISCUSSION THREAD

**Discussion by AhmadNasary** [spatie/laravel-backup#1242] [General]
i get this error

Backup failed because The dump process failed with exitcode 1 : General error : '"mysqldump"' is not recognized as an internal or external command,
operable program or batch file.
.
#0 E:\xammp\htdocs\sea\vendor\spatie\db-dumper\src\DbDumper.php(263): Spatie\DbDumper\Exceptions\DumpFailed::processDidNotEndSuccessfully(Object(Symfony\Component\Process\Process))
#1 E:\xammp\htdocs\sea\vendor\spatie\db-dumper\src\Databases\MySql.php(203): Spatie\DbDumper\DbDumper->checkIfDumpWasSuccessFul(Object(Symfony\Component\Process\Process), 'E:\\xammp\\htdocs...')
#2 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(252): Spatie\DbDumper\Databases\MySql->dumpToFile('E:\\xammp\\htdocs...')
#3 [internal function]: Spatie\Backup\Tasks\Backup\BackupJob->Spatie\Backup\Tasks\Backup\{closure}(Object(Spatie\DbDumper\Databases\MySql), 'mysql')
#4 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Support\Collection.php(638): array_map(Object(Closure), Array, Array)
#5 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(255): Illuminate\Support\Collection->map(Object(Closure))
#6 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(169): Spatie\Backup\Tasks\Backup\BackupJob->dumpDatabases()
#7 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Tasks\Backup\BackupJob.php(145): Spatie\Backup\Tasks\Backup\BackupJob->createBackupManifest()
#8 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Commands\BackupCommand.php(56): Spatie\Backup\Tasks\Backup\BackupJob->run()
#9 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(36): Spatie\Backup\Commands\BackupCommand->handle()
#10 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Container\Util.php(37): Illuminate\Container\BoundMethod::Illuminate\Container\{closure}()
#11 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(93): Illuminate\Container\Util::unwrapIfClosure(Object(Closure))
#12 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Container\BoundMethod.php(37): Illuminate\Container\BoundMethod::callBoundMethod(Object(Illuminate\Foundation\Application), Array, Object(Closure))
#13 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Container\Container.php(590): Illuminate\Container\BoundMethod::call(Object(Illuminate\Foundation\Application), Array, Array, NULL)
#14 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Console\Command.php(134): Illuminate\Container\Container->call(Array)
#15 E:\xammp\htdocs\sea\vendor\symfony\console\Command\Command.php(255): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#16 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Console\Command.php(121): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Illuminate\Console\OutputStyle))
#17 E:\xammp\htdocs\sea\vendor\spatie\laravel-backup\src\Commands\BaseCommand.php(16): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 E:\xammp\htdocs\sea\vendor\symfony\console\Application.php(1009): Spatie\Backup\Commands\BaseCommand->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 E:\xammp\htdocs\sea\vendor\symfony\console\Application.php(273): Symfony\Component\Console\Application->doRunCommand(Object(Spatie\Backup\Commands\BackupCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 E:\xammp\htdocs\sea\vendor\symfony\console\Application.php(149): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Console\Application.php(93): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 E:\xammp\htdocs\sea\vendor\laravel\framework\src\Illuminate\Foundation\Console\Kernel.php(131): Illuminate\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#23 E:\xammp\htdocs\sea\artisan(35): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#24 {main}
Sending notification failed
Backup failed because: The dump process failed with exitcode 1 : General error : '"mysqldump"' is not recognized as an internal or external command,
operable program or batch file.
.

In 7b19454b9250bfa9fda231b7d902f65d15760b99.php line 43:

  Undefined variable: subcopy (View: E:\xammp\htdocs\sea\resources\views\vendor\mail\html\layout.blade.php) (View: E:
  \xammp\htdocs\sea\resources\views\vendor\mail\html\layout.blade.php) (View: E:\xammp\htdocs\sea\resources\views\ven
  dor\mail\html\layout.blade.php)

In 7b19454b9250bfa9fda231b7d902f65d15760b99.php line 43:

  Undefined variable: subcopy (View: E:\xammp\htdocs\sea\resources\views\vendor\mail\html\layout.blade.php) (View: E:
  \xammp\htdocs\sea\resources\views\vendor\mail\html\layout.blade.php)

In 7b19454b9250bfa9fda231b7d902f65d15760b99.php line 43:

  Undefined variable: subcopy (View: E:\xammp\htdocs\sea\resources\views\vendor\mail\html\layout.blade.php)

In 7b19454b9250bfa9fda231b7d902f65d15760b99.php line 43:

  Undefined variable: subcopy

**Comment by freekmurze** [comment]
Probably you should take a look at subcopy in your own template
